content/docs/rfcs/10/README.md

```diff
@@ -133,6 +133,12 @@ For an experimental privacy-preserving economic spam protection mechanism, see [
 
 See [23/WAKU2-TOPICS](/spec/23) for more information about recommended topic usage.
 
+# Transport Protocol Specification
+
+This section describes the specification for transport protocols supported by waku v2. While Waku strives to be transport agonistic, however any underlying transport layer MUST be a reliable transport.
+
+Waku client implementations SHOULD support the TCP transport, and MUST be enabled for both dialing and listening even if other transports are available if TCP is available.
+The underlying transports SHOULD support secure websocket and MAY support websockets for bidirectional communication streams and web browser context.
```

Comment:
```suggestion
The underlying transports SHOULD support secure websocket and MAY support insecure websockets for bidirectional communication streams and web browser context.
```
following one of the @jm-clius's comment

Reply:
Maybe split into two sentences to make it clearer under what conditions `unsecure websockets` could be used.
Something like:
```suggestion
A Waku v2 node SHOULD support secure websockets for bidirectional communication streams, for example in a web browser context.
A node MAY support unsecure websockets if required by the application or running environment.
```

Reply:
done in ea014867f9062d531dd9f1772c99e4b2058f5f80